src/modules/balance/components/PaperWalletToDisplay.vue

```diff
@@ -136,7 +136,7 @@ export default {
   computed: {
     ...mapState('wallet', ['address', 'instance', 'isHardware']),
     key() {
-      if (this.address && !this.showPrivateKey) {
+      if (this.address && this.showPrivateKey) {
```

Comment:
There is this.instance.isPubOnly if that is true no private key you can use that for this if statement